### ISSUE TITLE
Performance benchmarks section of the README wrongly says we don't support `Distributed`

### ISSUE THREAD

**Issue by glwagner** [CliMA/Oceananigans.jl#3758]
cc @simone-silvestri 

https://github.com/CliMA/Oceananigans.jl/blob/3c95e7ee6772a2597e206ae35202500be1ef5b32/README.md?plain=1#L199

**Comment by simone-silvestri** [comment]
right, remaking the performance benchmark is definitely on the todo list!

**Comment by navidcy** [comment]
oh! let's change this phrase!

that's perhaps why a lot of people come and tell me "but it's not paralleled right?" and then I mention we run up to 764 GPUs and they stare at me with amazement...

**Comment by simone-silvestri** [comment]
Good idea, let's change the phrase but keep this issue open so that I remember to tackle the performance benchmarks

**Comment by simone-silvestri** [comment]
ah ok, I see it's a duplicate of #3684

**Comment by glwagner** [comment]
True though on #3684 I was referring to the plots, and didn't notice that there was actually wrong language
